whois-api/src/test/java/net/ripe/db/whois/api/ElasticSearchHelper.java

```diff
@@ -97,7 +96,7 @@ private XContentBuilder getSettings() throws IOException {
                         .startObject("my_email_analyzer")
                             .field("type", "custom")
                             .field("tokenizer", "uax_url_email")
-                            .field("filter", new String[]{"stop", "lowercase"})
+                            .field("filter", new String[]{"my_word_email_delimiter_graph", "lowercase", "english_stop"})
```

Comment:
by default stop uses _english_ so here english_stop is the same as stop

Reply:
Why change `stop` to `english_stop` then?

Reply:
I have been talking with Mahesh about it. We agree that for readability is better to keep english_stop